pandas-stubs/core/indexing.pyi

```diff
@@ -58,6 +58,10 @@ class _LocIndexerFrame:
     @overload
     def __getitem__(self, idx: _StrLike,) -> Series: ...
     @overload
+    def __getitem__(self, idx: List[_StrLike],) -> Series: ...
+    @overload
+    def __getitem__(self, idx: Tuple[_MaskType, Union[_MaskType, List[str]]]) -> DataFrame: ...
```

Comment:
isn't this one capture by

```python
def __getitem__(
        self, idx: Tuple[Union[_MaskType, List[str]], Union[_MaskType, List[str]]],
    ) -> DataFrame: ...
```
?

(see below)

Reply:
Yes, you're right

The reason it was added was because
```
@overload
def __getitem__(self, idx: Tuple[_StrLike, _StrLike],) -> float: ...
```

was capturing the case 
```
def __getitem__(self, idx: Tuple[_MaskType, Union[_MaskType, List[str]]]) -> DataFrame: ...
```

If the order of the overloads is changed, so that the order is now
```
@overload
def __getitem__(
    self, idx: Tuple[Union[_MaskType, List[str]], Union[_MaskType, List[str]]],
) -> DataFrame: ...
@overload
def __getitem__(self, idx: Tuple[_StrLike, _StrLike],) -> float: ...
```

then the underlying problem is dealt with